### PR TITLE
Expose grpc max inbound message size as cli option

### PR DESCRIPTION
**Pull request by petro-semeniuk-da**
`LedgerApiServer` is used by both index cli and sandbox cli so I had to put default into it.

Cli does not expose protocol specific (grpc) and have generic `maxInboundMessageSize` name.

### CODE REVIEW

File: ledger/sandbox/src/main/scala/com/digitalasset/ledger/server/apiserver/LedgerApiServer.scala [hunk 1]
```diff
@@ -31,9 +31,12 @@ trait ApiServer extends AutoCloseable {
 }
 
 object LedgerApiServer {
+  val DefaultMaxInboundMessageSize = 4194304
```
Comment (stefanobaghino-da) [inline ledger/sandbox/src/main/scala/com/digitalasset/ledger/server/apiserver/LedgerApiServer.scala]:
```suggestion
```

File: ledger/sandbox/src/main/scala/com/digitalasset/ledger/server/apiserver/LedgerApiServer.scala [hunk 2]
```diff
@@ -31,9 +31,12 @@ trait ApiServer extends AutoCloseable {
...
+
```
Comment (stefanobaghino-da) [inline ledger/sandbox/src/main/scala/com/digitalasset/ledger/server/apiserver/LedgerApiServer.scala]:
```suggestion
```

File: ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/config/SandboxConfig.scala [hunk 3]
```diff
@@ -49,6 +51,7 @@ final case class CommandConfiguration(
 object SandboxConfig {
 
   val DefaultPort = 6865
+  val DefaultMaxInboundMessageSize = LedgerApiServer.DefaultMaxInboundMessageSize
```
Comment (stefanobaghino-da) [inline ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/config/SandboxConfig.scala]:
```suggestion
  val DefaultMaxInboundMessageSize = 4194304
```

File: ledger/sandbox/src/main/scala/com/digitalasset/platform/index/config/Config.scala [hunk 4]
```diff
@@ -7,15 +7,20 @@ import java.io.File
...
     port: Int,
     portFile: Option[File],
     archiveFiles: List[File],
+    maxInboundMessageSize: Int,
     jdbcUrl: String,
     tlsConfig: Option[TlsConfiguration]
 )
 
 object Config {
+  val DefaultMaxInboundMessageSize = LedgerApiServer.DefaultMaxInboundMessageSize
```
Comment (stefanobaghino-da) [inline ledger/sandbox/src/main/scala/com/digitalasset/platform/index/config/Config.scala]:
```suggestion
  val DefaultMaxInboundMessageSize = 4194304
```

File: ledger/sandbox/src/main/scala/com/digitalasset/platform/index/config/Config.scala [hunk 5]
```diff
@@ -7,15 +7,20 @@ import java.io.File
...
+
```
Comment (stefanobaghino-da) [inline ledger/sandbox/src/main/scala/com/digitalasset/platform/index/config/Config.scala]:
```suggestion
```

File: ledger/sandbox/src/main/scala/com/digitalasset/platform/index/config/Config.scala [hunk 6]
```diff
@@ -7,15 +7,20 @@ import java.io.File
 
 import com.digitalasset.ledger.api.tls.TlsConfiguration
 
+import com.digitalasset.ledger.server.apiserver.LedgerApiServer
```
Comment (stefanobaghino-da) [inline ledger/sandbox/src/main/scala/com/digitalasset/platform/index/config/Config.scala]:
```suggestion
```

File: ledger/sandbox/src/main/scala/com/digitalasset/platform/index/config/Config.scala [hunk 7]
```diff
@@ -7,15 +7,20 @@ import java.io.File
...
+
```
Comment (stefanobaghino-da) [inline ledger/sandbox/src/main/scala/com/digitalasset/platform/index/config/Config.scala]:
```suggestion
```

File: ledger/sandbox/src/main/scala/com/digitalasset/platform/index/config/Config.scala [hunk 8]
```diff
@@ -7,15 +7,20 @@ import java.io.File
...
     jdbcUrl: String,
     tlsConfig: Option[TlsConfiguration]
 )
 
 object Config {
+  val DefaultMaxInboundMessageSize = LedgerApiServer.DefaultMaxInboundMessageSize
+
   def default: Config =
-    new Config(0, None, List.empty, "", None)
+    new Config(0, None, List.empty, DefaultMaxInboundMessageSize, "", None)
```
Comment (stefanobaghino-da) [inline ledger/sandbox/src/main/scala/com/digitalasset/platform/index/config/Config.scala]:
```suggestion
    new Config(0, None, List.empty, 4194304, "", None)
```

**Comment by raphael-speyer-da** [comment]
:tada:

**Pull request merged**